### PR TITLE
OCPBUGS-3313: Change the label name

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OCPBUGS-3313

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -108,7 +108,7 @@ With this release, there are several updates to the *Developer* perspective of t
 * Create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * Set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
 * Make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.
-* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes you to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes you to the *ResourceQuota details* page. 
+* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes you to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes you to the *ResourceQuota details* page.
```
Comment (arkadeepsen) [inline release_notes/ocp-4-12-release-notes.adoc:111]: This change is unrelated to the Bug. This can be avoided.

**Comment by xenolinux** [comment]
[PR](https://github.com/openshift/openshift-docs/pull/52528) is opened in the favor of this bug
